weave/trace/weave_client.py

```diff
@@ -859,13 +870,44 @@ def get_calls(
             columns: A list of columns to include in the response. If None,
                all columns are included. Specifying fewer columns may be more performant.
                Some columns are always included: id, project_id, trace_id, op_name, started_at
+            scored_by: A list of Scorers to filter by. Multiple scorers are ANDed together. You can
+                pass in just the name (fetching scores for all versions of the scorer) or the
+                full ref URI which will fetch scores for a specific version of the scorer.
 
         Returns:
             An iterator of calls.
         """
         if filter is None:
             filter = CallsFilter()
 
+        # This logic might be pushed down to the server soon, but for now it lives here:
+        if scored_by and len(scored_by) > 0:
```

Comment:
len clause seems redundant since empty list would be falsey

Reply:
fair